deploy/fb-submitter-chart/templates/network_policy.yaml

```diff
@@ -54,3 +54,20 @@ spec:
     ports:
     - protocol: TCP
       port: 3000
+---
+apiVersion: networking.k8s.io/v1
```

Comment:
What does this network policy do? Is this punching a hole out for the west-1 email? Or something else? I don't understand what the cidr is doing :(

Reply:
i just retested and seems to work without it, so i've removed it. I can't quite remember the reason why I added but it did at one stage not work without it